infrastructure/charts/monitoring/values.yaml

```diff
@@ -5,10 +5,10 @@
 replicaCount: 1
 
 image:
-  repository: ghcr.io/calgo-lab/green-db/monitoring
+  repository: registry.datexis.com/s40781/cache6
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag:
+  tag: latest
```

Comment:
Please do not override the tag, otherwise, it won't use the chart appVersion as tag.